lib/muffin_man/catalog_items/v20220401.rb

```diff
@@ -3,11 +3,16 @@
 module MuffinMan
   module CatalogItems
     class V20220401 < BaseApi
+      SEARCH_CATALOG_ITEMS_PARAMS = %w(
+        identifiers
+        identifiersType
+        sellerId
+      ).freeze
 
       API_VERSION = "2022-04-01".freeze
 
-      def search_catalog_items(keywords, marketplace_ids, params = {})
-        super(keywords, marketplace_ids, params, API_VERSION)
+      def search_catalog_items(marketplace_ids, params = {})
```

Comment:
This would be a breaking change. Might be worth updating version to 1.5 and noting the change as something that isn't backwards compatible in the CHANGELOG with details of what to update.

Reply:
Oh yeah, good call